src/utils/Utils.js

```diff
@@ -81,6 +82,43 @@ export const getRoundedTime = (roundedUp, offset) => {
   return date;
 };
 
+export const splitDayAndNightDataSets = (data) => {
+  let dayData = [];
+  let nightData = [];
+  data.forEach((d) => {
+    const hour = moment(d.date).hour();
+
+    if (hour > 5 && hour < 18) {
```

Comment:
Should use the values built into the dataset instead of hardcoded hours.  Those adjust for site location, time of year etc, to provide accurate day/night values.  This also probably is requiring a backend update to make sure this data is sent.

Reply:
yep! just quick hack to get it displayed. ideally should be done on the backend